components/checkoutForm.vue

```diff
@@ -40,6 +40,7 @@
         </li>
       </ul>
     </div>
+    <email-dialog ref="emailDialog" :emailRef="'in-bundle'" />
```

Comment:
I think it is better to place the email dialog inside bundle page, as the child component should know nothing about its container

Reply:
then the email checking logic (or even the checkout logic?) should be moved too?
or just fire event from inside and trigger `email-dialog` accordingly ?

Reply:
Maybe pass `emailRef` as props first